packages/bsync/tests/operations.test.ts

```diff
@@ -16,6 +16,14 @@ describe('operations', () => {
   let bsync: BsyncService
   let client: BsyncClient
 
+  const validPayload0 = Buffer.from(
+    new TextEncoder().encode(JSON.stringify({ value: 0 })),
+  )
+  const validPayload1 = Buffer.from(
+    new TextEncoder().encode(JSON.stringify({ value: 1 })),
+  )
+  const invalidPayload = Buffer.from(new TextEncoder().encode('{invalid json}'))
```

Comment:
Would using e.g. `Buffer.from('{invalid json}', 'utf8')` do the trick?  (utf8 is the default so technically that second arg is not needed, but would be a good way to be explicit if you like.)